src/components/featureHighlight/index.tsx

```diff
@@ -298,7 +332,7 @@ class FeatureHighlight extends BaseComponent {
               titleStyle
             ]}
             numberOfLines={titleNumberOfLines}
-            pointerEvents="none"
+            pointerEvents
```

Comment:
I don't understand the meaning of this change. `pointerEvets` prop is not a boolean so why did you drop the value ("none")?

Reply:
It becomes boolean from out modifiers, and some how react-native haven't export this prop with a type, 
so I changed it back and added `@ts-expect-error`